contributors/devel/cherry-picks.md

```diff
@@ -25,17 +25,19 @@ task is for backporting PRs from master to release branches.
    `upstream/release-3.14`: `hack/cherry_pick_pull.sh upstream/release-3.14
    98765`
  * Your cherrypick PR targeted to the release branch will immediately get the
-   `do-not-merge/cherry-pick-not-approved` label. The release branch owner
+   `do-not-merge/cherry-pick-not-approved` label. The Patch Release Manager
    will triage PRs targeted to the branch.  Normal rules apply for code merge.
    * Reviewers `/lgtm` and owners `/approve` as they deem appropriate.
-   * The approving release branch owner is responsible for applying the
-     `cherrypick-approved` label.
+   * Milestone approval needs to be added for the target release branch.
```

Comment:
What does this mean?  Someone needs to apply `status/approved-for-milestone`? Who would that be?

Reply:
Is it correct as follows?

```
* Milestone label must be applied to match the target release branch
* SIG leads associated with the PR must indicate `/status approved-for-milestone`
```

Or does the milestone labeling happen via the cherry pick script?

Reply:
AFAIK, they do not. The two sample cherry-picks I saw did not have them automatically included; they were added manually by the reviewer.
I was going for something like:

Normal rules apply for code merge, such as:
- reviews and lgtms
- milestones and approvals
- passing tests
- hearts and smilies

and then focus on what's *different* in cherry picks.

Reply:
So I feel like this is ok then?  It says "Normal rules apply for code merge" and milestone approval requirements are maybe hopefully sort of going to appear in contributors/devel/issues.md in my https://github.com/kubernetes/community/pull/2430?  But now that I say that here and have posted small updates there, I wonder if that file should not be issues.md but rather something like milestones.md since it's trying to cover issues and PRs?

Reply:
> SIG leads associated with the PR must indicate `/status approved-for-milestone`

This is only required during code slush. Otherwise, this label is not really used: https://github.com/kubernetes/kubernetes/pulls?q=is%3Apr+label%3Acherrypick-approved+is%3Aclosed.

Reply:
Follow up question: when is `cherrypick-candidate` used and how is it different from `cherrypick-approved`? There does not seem to be any consistency in how this label is being used right now.

Reply:
I assumed `cherrypick-candidate` was a nonnecessary pre-stage for `cherrypick-approved`?

or perhaps it is related to the release cycle?

Reply:
I looked for labels in k/k and cherrypick-candidate is obsolete. Out it goes.